src/sidebar/components/MenuItem.tsx

```diff
@@ -239,7 +239,7 @@ export default function MenuItem({
   const wrapperClasses = classnames(
     'focus-visible-ring ring-inset',
     'w-full min-w-[150px] flex items-center select-none',
-    'border-b',
+    'border-b rounded-none cursor-pointer',
```

Comment:
This addresses inconsistencies in which link menu items were being rendered with rounded corners on hover, while non-links were not rounded.

I took the opportunity to address other inconsistencies between link vs non-link menu items, like hover color and cursor type.